Dockerfile-debian-core.template

```diff
@@ -0,0 +1,13 @@
+FROM debian:name-slim
+
+RUN groupadd --gid 1000 node \
+  && useradd --uid 1000 --gid node --shell /bin/bash --create-home node
+
+ENV NODE_VERSION 0.0.0
+
+COPY --from=node:0.0.0-slim /usr/local/bin/ /usr/local/bin/
```

Comment:
This creates some dangling symlinks, but maybe they are fine? 🤷 

Would it be better if the images were built in the other direction so that they can share the node installation layers?  i.e. `node:slim` is built `FROM node:core`. Or maybe an approach like https://github.com/docker-library/golang/pull/505 to clamp timestamps and use `COPY --link` to get more layer sharing?

Reply:
@yosifkit does `--link` for part of a layer?

Reply:
Added the `--link`